x-pack/plugins/cross_cluster_replication/index.js

```diff
@@ -15,10 +15,30 @@ export function crossClusterReplication(kibana) {
     configPrefix: 'xpack.cross_cluster_replication',
     publicDir: resolve(__dirname, 'public'),
     require: ['kibana', 'elasticsearch', 'xpack_main'],
+
     uiExports: {
       styleSheetPaths: `${__dirname}/public/index.scss`,
       managementSections: ['plugins/cross_cluster_replication'],
+      injectDefaultVars(server) {
+        const config = server.config();
+        return {
+          ccrUiEnabled: config.get('xpack.cross_cluster_replication.ui.enabled'),
```

Comment:
Is the corresponding ES config prefix spelled out or is it ccr?  I think we should keep these aligned.